gravatar/src/main/java/com/gravatar/services/AuthenticationInterceptor.kt

```diff
@@ -0,0 +1,20 @@
+package com.gravatar.services
+
+import okhttp3.Interceptor
+import okhttp3.Request
+import okhttp3.Response
+import com.gravatar.di.container.GravatarSdkContainer.Companion.instance as GravatarSdkDI
+
+internal class AuthenticationInterceptor : Interceptor {
```

Comment:
I pushed a quick modification in order to improve readability and facilitate adding other headers if needed in the future. Tell me what you think.